server/graphql/v2/interface/Account.js

```diff
@@ -29,6 +29,120 @@ import { NotFound } from '../../errors';
 
 import models, { Op } from '../../../models';
 
+const accountFieldsDefinition = () => ({
```

Comment:
What is the purpose of this change?

Reply:
I noticed a ton a repetition on fields that didn't need a special resolver. For example:
```
  id: {
    type: GraphQLString,
    description: 'The public id identifying the account (ie: 5v08jk63-w4g9nbpz-j7qmyder-p7ozax5g)',
  },
  slug: {
    type: GraphQLString,
    description: 'The slug identifying the account (ie: babel)',
  },
  type: {
    type: AccountType,
    description: 'The type of the account (BOT/COLLECTIVE/EVENT/ORGANIZATION/INDIVIDUAL)',
  },
  name: {
    type: GraphQLString,
  },
  description: {
    type: GraphQLString,
  },
  website: {
    type: GraphQLString,
  },
  twitterHandle: {
    type: GraphQLString,
  },
  githubHandle: {
    type: GraphQLString,
  },
  currency: {
    type: GraphQLString,
  },
  imageUrl: {
    type: GraphQLString,
    args: {
      height: { type: GraphQLInt },
      format: {
        type: ImageFormat,
      },
    },
  },
  createdAt: {
    type: GraphQLDateTime,
    description: 'The time of creation',
  },
  updatedAt: {
    type: GraphQLDateTime,
    description: 'The time of last update',
  },
```
Width this change we can avoid repeating them in both places in the same file. I noticed this in the `Transaction.js` interface too but, I preferred to  wait for you opinion before refactoring more files.

Reply:
But the resolver is missing now. Are you saying it's optional in these simple cases?

Reply:
Yeah, if we are returning from the db an object like :`{ id: 1, description: "..." }` and the field names are `id` and `description` the engine can resolve them with the values from the object. It is when we have custom logic that we need the resolver function.

Reply:
It's great to know, never realized that!